python/action-server/main.py

```diff
@@ -0,0 +1,154 @@
+import time
+import json
+import itertools
+import zenoh
+from zenoh import Reliability, SampleKind, Query, Sample, KeyExpr, QueryTarget, Value
+from pydantic import BaseModel
+import logging
+import asyncio
+import yaml
+import state_machine
+log = logging.getLogger(__name__)
+
+# describe settings for parsing values.
+class ActionSettings(BaseModel):
+    mode: str = None
+    connect: str = None
+    listen: str = None
+    config: str = None
+    base_key_expr: str
+    iter: int
+    target: str
+    declare_key_expr: str
+
+store = {}
```

Comment:
what does this do?

Reply:
This stores the queries of the queryables to response.

Reply:
Deleted this